lib/widgets/components/tickets/swipe_overlay.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:coffeecard/cubits/tickets/tickets_cubit.dart';
```

Comment:
This overlay needs to be changed.

Wrong scrim color
Use of magic number for height
Reliance on device width for slider width

Reply:
#213